### PR TITLE
Fixes Lint to Ignore Certain Files

### PR DESCRIPTION
**Pull request by seemavora**
Resolves Issue #495

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+api/node_modules/
+api/printingRPC/
+api/config/
+src/config/
+node_modules/
+printingRPC/
+build/
```
Comment (hsorathia) [inline .eslintignore:8]: new line at the end of file

File: api/SceHttpServer.js [hunk 2]
```diff
@@ -88,10 +88,12 @@ class SceHttpServer {
         useCreateIndex: true
       })
       .then(() => {
-        console.log('MongoDB Connection Successful')
-        console.log()
+        // console.log('MongoDB Connection Successful')
```
Comment (hsorathia) [inline api/SceHttpServer.js:91]: remove comments

File: package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
     "frontend": "react-scripts start",
     "build": "react-scripts build",
     "deploy": "git pull && npm install && pm2 stop database api app && pm2 delete database api app && pm2 start mongod --name \"database\" && npm run build && pm2 start npm --name \"app\" -- start && pm2 start npm \"app\" -- run dev",
-    "lint": "./node_modules/.bin/eslint src/ api/models/ api/mailer/ api/routes/ api/util util/ test/"
+    "lint": "./node_modules/.bin/eslint ./"
```
Comment (hsorathia) [inline package.json:14]: nothing to change here but this is cool, I didn't know that it could work like this

File: .eslintignore (hunk 1 above)
Comment (evanugarte) [reply .eslintignore:8]: yup

File: api/SceHttpServer.js (hunk 2 above)
Comment (evanugarte) [reply api/SceHttpServer.js:91]: you need to keep the print statements here.
just change them to console.debug

File: .eslintignore [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (evanugarte) [inline .eslintignore:1]: remove this blank line at the top

**Pull request merged**